sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java

```diff
@@ -16,8 +16,8 @@
 import io.opentelemetry.api.common.AttributesBuilder;
 import io.opentelemetry.api.internal.StringUtils;
 import io.opentelemetry.api.internal.Utils;
+import io.opentelemetry.sdk.common.OtelVersion;
```

Comment:
My instinct is to make this internal. Any reason why you didn't?

Reply:
Just seemed unnecessary to protect a single constant string....but I suppose the concern is that users could start using it in unexpected/creative ways that would force maintainers to never be able to remove it? 

I can put it into an "internal" package...lemme try that.

Reply:
Ok, added "internal" path + package component.